### PR TITLE
Unbreak configuration of a VLAN from iBFT

### PR DESCRIPTION
**Pull request by lkundrak**

### CODE REVIEW

**Comment by lkundrak** [comment]
Ping?

File: modules.d/40network/net-lib.sh [hunk 1]
```diff
@@ -263,10 +263,12 @@ ibft_to_cmdline() {
                     case "$vlan" in
                         [0-9]*)
                             echo "vlan=$dev.$vlan:$dev"
+                            dev="$dev.$vlan"
                             echo $mac > /tmp/net.${dev}.${vlan}.has_ibft_config
```
Comment (haraldh) [inline modules.d/40network/net-lib.sh]: Wouldn't that change:
 echo $mac > /tmp/net.${dev}.${vlan}.has_ibft_config
to
 echo $mac > /tmp/net.${dev}.${vlan}.${vlan}.has_ibft_config

effectively?

**Comment by danimo** [comment]
Ping?

**Comment by haraldh** [comment]
ping @lkundrak

**Comment by haraldh** [comment]
ping @lkundrak

File: modules.d/40network/net-lib.sh (hunk 1 above)
Comment (tblume) [reply modules.d/40network/net-lib.sh:267]: Is a command line  like "ip=ibft0.666:dhcp" allowed at all?
That would mean reconfiguring the interface that was initially set up via firmware.
If the original ip setup from firmware gets overwritten by dhcp, is the interface still to be called ibft?

**Pull request closed without merging**